In [73]:
import pandas as pd

df = pd.read_csv('scores/handle_time_score.csv', index_col=0)
df

old        prophet          kpi start_train   end_train
bro   48697.394231   43695.842907  handle_time  2016-03-10  2020-06-01
psg   82164.500000   73158.413740  handle_time  2017-01-10  2020-06-01
wis  497281.224500  393280.911400  handle_time  2016-01-20  2020-06-01
scs  124690.785700  100969.886000  handle_time  2018-01-20  2020-06-01
ris  141385.193900  112769.400400  handle_time  2017-01-20  2020-06-01
rcs   37143.693880   37131.786030  handle_time  2016-04-30  2020-06-01
col  124690.785714  484255.237171  handle_time  2017-01-01  2020-06-01

In [71]:
import os
end = ['2020-06-01']*7
start = ['2016-03-10','2017-01-10', '2016-01-20', '2017-01-01', 
       '2018-01-20', '2017-01-20', '2016-04-30']
bu = ['bro', 'psg', 'wis', 'col', 'scs', 'ris', 'rcs']
kpi = ['handle_time']*7



In [74]:
# for line in list(zip(bu, kpi, start, end)):
#     stream = os.popen('python code/run_model.py '+line[0]+'_aht_vol.csv '+line[1]+ ' '+line[2]+' '+line[3]+ ' 200')
#     output = stream.readlines()
#     output

In [75]:
df['diff (sec)'] = (df['old'] - df['prophet'])

In [76]:
df['diff (hours)'] = df['diff (sec)']/3600

In [77]:
hours_saved = df[df.index!='col']['diff (hours)'].sum()
hours_saved

47.321264370544434

In [78]:
hours_saved * 262 * 25

309954.281627066

# approximately 47 hours per day in handle time savings
accross 262 business days at approximately `$25 per hour = $309,954 per year` in labor reallocation cost savings across BRO, PSG, RIS, SCS, RCS, and WISE 